parkings/models/region.py

```diff
@@ -12,11 +12,25 @@
 class RegionQuerySet(models.QuerySet):
     def with_parking_count(self, at_time=None):
         time = at_time if at_time else timezone.now()
-        valid_parkings_q = (
-            Q(parkings__time_start__lte=time) &
-            (Q(parkings__time_end__gte=time) | Q(parkings__time_end=None)))
-        return self.annotate(
-            parking_count=Count(Case(When(valid_parkings_q, then=1))))
```

Comment:
Please don't change the formatting of the existing code. It seems that you didn't touch the functionality of this code at all.

Reply:
The point of this change is to use the ActiveParkings model to get the forecasted parking counts for future datetime queries:
```
        if time > timezone.now():
            region_q = self.filter(
                active_parkings__time=time, active_parkings__is_forecast=True
            )
```
or the pre-calculated active parking count for past datetime queries:
```
        else:
            region_q = self.filter(
                active_parkings__time=time, active_parkings__is_forecast=False
            )
```
if there exists an ActiveParking instance for that datetime.

Should I separate this to its own QuerySet method perhaps?